src/styles/components/tabs.scss

```diff
@@ -4,28 +4,65 @@
 .tabs {
   &__header {
     width: 100%;
-    white-space: nowrap;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
     overflow-x: auto;
-    border-bottom: 0.0625rem solid $colour-platinum;
+    box-shadow: inset 0 -0.1rem 0 0 $colour-platinum;
     margin-bottom: $global-margin;
+    color: $colour-sapphire-blue;
+    font-weight: 600;
 
     &__item {
-      display: inline-block;
+      flex: 0 0 auto;
       padding: $global-padding;
       cursor: pointer;
       -webkit-user-select: none;
       -moz-user-select: none;
       -ms-user-select: none;
       user-select: none;
 
+      &--bordered {
+        border-right: 0.1rem solid $colour-platinum;
+        border-top: 0.1rem solid $colour-platinum;
+        border-radius: 0.2rem 0.2rem 0 0;
+        background-color: initial;
+
+        &:first-child {
+          border-left: 0.1rem solid $colour-platinum;
+        }
+      }
+
+      &--active,
       &:hover,
       &:focus {
-        color: $colour-sea-blue;
+        background-color: rgba($colour-pastel-blue, 0.19);
```

Comment:
And in this case, if you want to explore new CSS features, have a look at this page https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color#relative_value_syntax
I think this would end up being `background-color: color(from var(--fr--color-pastel-blue) srgb r g b / 0.19);`.

Support is fairly good now https://caniuse.com/css-color-function but if you want you can provide a simpler fallback colour in the line before in case it's not supported